frontend/src/app/main/ui/workspace/sidebar/options/menus/shadow.scss

```diff
@@ -53,14 +55,17 @@
 }
 
 .basic-options {
-  @include flexRow;
+  display: grid;
+  grid-template-columns: repeat(8, var(--sp-xxxl));
+  gap: var(--sp-xs);
 }
 
 .shadow-info {
+  grid-column: span 6;
   display: flex;
   align-items: center;
   gap: $s-1;
-  width: $s-188;
+  width: 100%;
```

Comment:
if we're setting the `grid-column` above, we shouldn't need this `width` (and if we do, let's change the `display: flex` for a `display: grid`, which sets an implicit width for the element.